resources/styleheets/components/_cards.scss

```diff
@@ -0,0 +1,78 @@
+
+.box {
+    padding: 20px;
+    border-radius: 25px; 
+    border: 2px solid rgba(255, 255, 255, 0.5); 
+    max-width: 80%;
+    color: $font_w;
+    text-align: $ali_center;
+    margin: auto;
+}
+
+.div{
+    display: $fila;
+    align-items: $ali_center;
+    padding: $p1;
+    border-top: 1px $sol $font_w1; 
+}
+.user{
+    display: $fila;
+    align-items: $ali_center;
+    font-size: $p2 ;
+    .profile{
+        cursor: $poit;
+        transition: background-color 0.3s, color 0.3s; 
+        &:hover {
+            color: $theme_blue;       
+        }
+    }
+}
+
+.date{
+    font-size: $p1_5;
+    margin-left: $p1;
+    color: $font_w1;
+}
+.text{
+    text-align: $ali_left;
+    p{
+        font-size: 18px ;
+    }
+}
+.hidden-important {
+    display: $no !important;
+}
+@media (max-width: 800px) {
+    .hidden {
+        display: $no;
+    }
+}
+@media (min-width: 800px){
+    .hidden_{
```

Comment:
useless declaration of classes, you don't have to make classes for each element you have.
it is possible refactorized as
```
// if you make the changes above you can use the @media as @include in every file
@include desktop-xl {
  .text {
    p {
      display: none;
    }
  }
}
```
and if you need the exact first p element just select with `::first-child` selector.
[here](https://www.w3schools.com/cssref/css_selectors.php) is a cheatsheet for it, remember do not declare classes for just one element, you can select it even with other forms without the class or id